healthcare/v1/src/main/java/snippets/healthcare/dicom/DicomStoreList.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019 Google LLC
```

Comment:
If it's a new file, 2020.

Reply:
The OSS team said that if you're just moving a file, then you keep the original copyright date. I don't know the particulars of whether copying a file into a new directory constitutes "moving", wdyt?